pkg/arvo/neo/cod/std/src/imp/counter.hoon

```diff
@@ -36,11 +38,23 @@
     ::
     ::  de-vase the state; we don't know what it is,
     ::  in most cases it will be counter's old state
-    +*  this  !<(number state-vase)
+    +*  state  !<(number state-vase)
```

Comment:
let's get rid of the `+*` here and do it with a `=/` in the on-poke arm instead.